src/check_update.c

```diff
@@ -34,6 +34,7 @@
 
 static void print_help(void)
 {
+	print("Checks whether an update is available and prints out the information if so\n\n");
```

Comment:
I would quote this as

Checks whether an update is available along with current & server version

Reply:
Well, to check if there is an update available it needs to check the current version and the server version. So it kind of seems redundant to me that way.

Reply:
hmm thinking of it makes sense